tests/test_cases/test_sv_interface/Makefile

```diff
@@ -0,0 +1,14 @@
+# This file is public domain, it can be freely copied without restrictions.
+# SPDX-License-Identifier: CC0-1.0
+
+TOPLEVEL_LANG ?= verilog
+VERILOG_SOURCES = $(shell pwd)/top.sv
+MODULE = test_sv_if
+TOPLEVEL = top
+
+ifneq ($(shell echo $(SIM) | tr A-Z a-z),verilator)
```

Comment:
Allow this test to run with Riviera and Questa. All verilator tests are allowed to fail since it's broken upstream rn, so this isn't being tested.

Reply:
Also Xcelium/IUS and VCS I guess.